### PR TITLE
Update table scan_report and extract cvss_v3_score from vendor attribute

### PR DESCRIPTION
**Pull request by stonezdj**
For better performance when query cve information, add summary information to scan_report
    Extract cve_score from vendor attribute in vulnerability_record
    SQL migrate script for the update

Thank you for contributing to Harbor!

# Comprehensive Summary of your change

# Issue being fixed
Fixes #(issue)

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [x] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

File: src/pkg/scan/postprocessors/report_converters.go [hunk 1]
```diff
@@ -290,3 +297,74 @@ func toVulnerabilityItem(record *scan.VulnerabilityRecord, artifactDigest string
...
+			FixableCnt++
+		}
+	}
+
+	reports, err := report.Mgr.List(ctx, q.New(q.KeyWords{"uuid": reportUUID}))
+	if err != nil {
+		return err
+	}
+	if len(reports) == 0 {
+		return fmt.Errorf("report not found, uuid:%v", reportUUID)
```
Comment (wy65701436) [inline src/pkg/scan/postprocessors/report_converters.go:338]: please use errors.New().WithMessage()

File: src/pkg/scan/postprocessors/report_converters.go [hunk 2]
```diff
@@ -290,3 +297,74 @@ func toVulnerabilityItem(record *scan.VulnerabilityRecord, artifactDigest string
 
 	return item
 }
+
+// updateReport updates the report summary with the vulnerability counts
+func (c *nativeToRelationalSchemaConverter) updateReport(ctx context.Context, vulnerabilities []*vuln.VulnerabilityItem, reportUUID string) error {
+	log.Debugf("Update report summary for report, uuid %v", reportUUID)
```
Comment (wy65701436) [inline src/pkg/scan/postprocessors/report_converters.go:303]: log.G(ctx).WithFields(fields).Errorf("Update report summary for report, uuid %v", reportUUID)

File: src/pkg/scan/postprocessors/report_converters.go [hunk 3]
```diff
@@ -290,3 +297,74 @@ func toVulnerabilityItem(record *scan.VulnerabilityRecord, artifactDigest string
...
+}
+
+// Nvd ...
+type Nvd struct {
+	V3Score float64 `json:"V3Score"`
+}
+
+func parseScoreFromVendorAttribute(vendorAttribute string) (NvdV3Score float64) {
+	var data map[string]CVSS
+	_ = json.Unmarshal([]byte(vendorAttribute), &data)
```
Comment (wy65701436) [inline src/pkg/scan/postprocessors/report_converters.go:365]: please don't ignore error

**Review by wy65701436 — APPROVED** [review]
lgtm

File: src/pkg/scan/postprocessors/report_converters.go (hunk 1 above)
Comment (stonezdj) [reply src/pkg/scan/postprocessors/report_converters.go:338]: Updated

File: src/pkg/scan/postprocessors/report_converters.go (hunk 2 above)
Comment (stonezdj) [reply src/pkg/scan/postprocessors/report_converters.go:303]: Updated

File: src/pkg/scan/postprocessors/report_converters.go (hunk 3 above)
Comment (stonezdj) [reply src/pkg/scan/postprocessors/report_converters.go:365]: Updated

**Review by chlins — APPROVED** [review]
lgtm

**Pull request merged**